### PR TITLE
curvefs/client: rm create fs in init

### LINKED ISSUE #891: curvefs/tool: create-fs
**Is your feature request related to a problem? (你需要的功能是否与某个问题有关?)**

use curvefs_tool to create fs

**Describe the solution you'd like (描述你期望的解决方法)**

command like:
curvefs_tool to create fs -fsName=/test

**Describe alternatives you've considered (描述你想到的折衷方案)**

**Additional context/screenshots (更多上下文/截图)**

### PR DESCRIPTION
**Pull request by Cyber-SiKu**
1. curvefs_tool has add create-fs
2. fix some error output

### What problem does this PR solve?

Issue Number: close #891  

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by YunhuiChen** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by YunhuiChen** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

File: curvefs/src/client/fuse_client.cpp [hunk 1]
```diff
@@ -151,15 +151,9 @@ CURVEFS_ERROR FuseClient::FuseOpInit(void *userdata,
...
-            LOG(INFO) << "The fsName not exist, try to CreateFs"
-                      << ", fsName = " << fsName;
-
-            CURVEFS_ERROR ret2 = CreateFs(userdata, &fsInfo);
-            if (ret2 != CURVEFS_ERROR::OK) {
-                LOG(ERROR) << "CreateFs failed, ret = " << ret2
-                           << ", fsName = " << fsName;
-                return ret2;
-            }
+            LOG(ERROR) << "The fsName not exist, try to CreateFs"
```
Comment (xu-chaojie) [inline curvefs/src/client/fuse_client.cpp:154]: ”try to CreateFs“ is not need.

File: curvefs/src/tools/create/curvefs_create_fs.cpp [hunk 2]
```diff
@@ -137,8 +144,9 @@ bool CreateFsTool::AfterSendRequestToHost(const std::string& host) {
                      << ", error text: " << controller_->ErrorText()
                      << std::endl;
         ret = false;
-    } else if (response_->statuscode() == mds::FSStatusCode::OK) {
-        // create success
+    } else if (response_->statuscode() == mds::FSStatusCode::OK ||
+               response_->statuscode() == mds::FSStatusCode::FS_EXIST) {
```
Comment (xu-chaojie) [inline curvefs/src/tools/create/curvefs_create_fs.cpp:148]: maybe fs is truely exist, still return success?

Comment (Cyber-SiKu) [reply curvefs/src/tools/create/curvefs_create_fs.cpp:148]: > maybe fs is truely exist, still return success?

This is a bug that was fixed by other PRs, and I forgot to modify it here.

File: curvefs/src/tools/create/curvefs_create_fs.cpp [hunk 3]
```diff
@@ -137,8 +144,9 @@ bool CreateFsTool::AfterSendRequestToHost(const std::string& host) {
...
+        // create success or fs exist
         std::cout << "create fs success." << std::endl;
     } else if (response_->statuscode() == mds::FSStatusCode::FS_EXIST) {
```
[the remark this answers is not in the document]
Comment (Cyber-SiKu) [reply curvefs/src/tools/create/curvefs_create_fs.cpp:151]: This is a bug that was fixed by other PRs, and I forgot to modify it here.

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
reckeck

**Comment by Cyber-SiKu** [comment]
reckeck

**Comment by Cyber-SiKu** [comment]
reckeck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Review by xu-chaojie — APPROVED** [review]

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Review by wu-hanqing — APPROVED** [review]

**Pull request merged**